src/clr-angular/utils/drag-and-drop/providers/drag-and-drop-event-bus.spec.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {ClrDraggableEvent} from "../interfaces/draggable-event";
+
+import {ClrDragAndDropEventBus} from "./drag-and-drop-event-bus";
+
+export default function(): void {
+    describe("Drag And Drop Event Bus Provider", function() {
+        beforeEach(function() {
+            this.dndEventBus = new ClrDragAndDropEventBus();
+        });
+
+        it("notifyAllOnDragStart() should cast to dragStart subject", function() {
```

Comment:
exposes an Observable to follow `dragStart` events?

would this be better? check with @youdz because I am terrible at this.

same for all messages below.

Reply:
`it("exposes an Observable to follow dragStart events")` seems better to me, you're right.